### PR TITLE
Add location tracking

### LINKED ISSUE #934: Log location coordinates in the background
> As a supervisor, I would like to collect location coordinates in the background to provide evidence that data was collected in a particular place.

It's an option we are going to implement in ODK Collect.
Issue: https://github.com/opendatakit/collect/issues/2773
Pull request: https://github.com/opendatakit/collect/pull/2772

Once it's done we should describe the change in the doc.

### PR DESCRIPTION
**Pull request by yanokwa**
Closes #934

#### What is included in this PR?
* Removed unnecessary commas and spaces
* Added a section on location tracking that describes how it's enabled and the implementation
* Added example output with explanations of columns
* Updated table with new events

**Reviewers**
Please pay close attention to description of location-priority values, the description of how the implementation works, and the description of the new events. If there are recommended values for the various variables, please suggest them.

### CODE REVIEW

File: odk1-src/form-audit-log.rst [hunk 1]
```diff
@@ -30,12 +30,50 @@ Enabling audit logging
...
+  `high-accuracy`: The most accurate location provided by the device, regardless of power use.
+
+  `balanced`: Block level accuracy (~100 meters). Uses less power than :tc:`high-accuracy`.
+
+  `low-power`: City level accuracy (~10 kilometers). Uses less power than :tc:`balanced`.
+
+  `no-power`: No locations will be returned unless another application on the device has requested location updates. Uses no additional power.
+
+:tc:`location-min-interval`
+  The desired minimum time, in seconds, location updates will be fetched by the device.
```
Comment (zestyping) [inline odk1-src/form-audit-log.rst]: ```...minimum interval, in seconds, at which location updates...```

File: odk1-src/form-audit-log.rst [hunk 2]
```diff
@@ -30,12 +30,50 @@ Enabling audit logging
...
+
+:tc:`location-max-age`
+  The maximum time, in seconds, locations will be considered valid by the device. Must be greater than or equal to :tc:`location-min-interval`.
+
+.. csv-table:: survey
+  :header: type, name, parameters
+
+  audit, audit, location-priority=balanced location-min-interval=60 location-max-age=120
+
+When location tracking is enabled, ODK Collect requests location updates from Android at the :tc:`location-min-interval`. The requests are sent with :tc:`location-priority` to ensure Android does not use more power than is desired.
```
Comment (zestyping) [inline odk1-src/form-audit-log.rst]: Maybe for clarity?

```
... from Android periodically, with an interval determined by :tc:`location-min-interval` ...
```

File: odk1-src/form-audit-log.rst [hunk 3]
```diff
@@ -73,44 +112,75 @@ Values in the :th:`node` column represent the node in the form that the event re
...
+  question,/data/age,1550615097082,1550615097655,37.4229983,-122.084,14.086999893188477
+  question,/data/name,1550615097656,1550615102351,37.4229983,-122.084,14.086999893188477
+  location tracking enabled,,1550615099271,,37.4229983,-122.084,14.086999893188477
+  question,/data/age,1550615102351,1550615107630,37.4229983,-122.084,14.086999893188477
+  end screen,,1550615107631,1550615109199,37.4229983,-122.084,14.086999893188477
+  form save,,1550615109199,,37.4229983,-122.084,14.086999893188477
+  form exit,,1550615109199,,37.4229983,-122.084,14.086999893188477
+  form finalize,,1550615109199,,37.4229983,-122.084,14.086999893188477
+
+Values in the :th:`latitude` and :th:`longitude` column represent the latitude and longitude in decimal degrees. Values in the :th:`accuracy` column represents accuracy in seconds.
```
Comment (zestyping) [inline odk1-src/form-audit-log.rst]: `column` -> `columns`

**Comment by yanokwa** [comment]
Thanks for the review! I've made the changes.

**Pull request merged**